app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,24 @@
 <html>
   <head>
     <title>Dripbox</title>
+
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+
+
   </head>
 
   <body>
+    <%unless current_page?('/')%><!-- Header soll nicht in Registration Log in auftauchen -> Links für die beiden Seiten hier noch einzufügen-->
+      <%=render "layouts/_header" %>
+    <%end%>
```

Comment:
Dafür gibt es bereits ein Feature von Rails. Das baust du gerade hier nach.

https://edgeguides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method

Reply:
ich weiß nicht was hier gemeint ist. das unless oder das rendern
und es sind doch nur 2 zeilen code. Wieso soll ich dafür unbedingt ein Rails Feature nutzen?

Reply:
Weil man das Framework lernen sollte und nicht alles direkt falsch machen. Um es genau zu sagen :).
Heute 2 Zeilen, morgen noch 2. Am Ende ist das ganze Projekt nicht mehr wartbar. Vor allem dann nicht, wenn jemand dann reinkommt, der Rails gut kennt. Dann versteht er nichts mehr. Er wartet, dass die Features von Rails benutzt werden.

Reply:
Gemeint ist immer das, was in der Nachricht oben zu sehen ist. Also alle 3 Zeilen.